modules/apps/account/account-service/service.xml

```diff
@@ -63,6 +63,9 @@
 
 		<!-- Finder methods -->
 
+		<finder name="A" return-type="Collection">
+			<finder-column name="accountEntryId" />
```

Comment:
I just realized that we don't scope the results to company. Can we safely assume that each accountEntryId is unique across different companies? If so, maybe we can remove the `companyId` field from the `AccountEntryUserRel` table. If not, we'll need to take into account of companyId in all the finders.